api/cmd/api/main.go

```diff
@@ -78,15 +81,28 @@ func main() {
 		logger.Fatalf("Failed to connect to the blockchain provider, error: %v", err)
 	}
 
+	awsSession, err := session.NewSession(&aws.Config{
+		Region: aws.String(cfg.Region),
+	})
+	if err != nil {
+		logger.Fatalf("Failed to create aws session: %v", err)
+	}
+	s3Client := s3.NewS3Client(awsSession)
+
 	walletMerkleProofRepository := repository.NewWalletMerkleProofDbImpl(db)
 	nftTokenRepository := repository.NewNftTokenDbImpl(db)
 	nftCollectionRepository := repository.NewNftCollectionDbImpl(db)
 	storyChapterRepository := repository.NewStoryChapterDbImpl(db)
 	storyInfoRepository := repository.NewStoryInfoDbImpl(db)
-
-	storyContentRepository, err := repository.NewStoryContentFsImpl(cfg.ContentPath)
+	/*
```

Comment:
do you want to keep the fs implementation?  you may add an option, or based on the URI to choose the repo. Please add some comment here.

Reply:
Will keep the fs implemention in case we want to use in the future. But remove the commented out section here